qt-bugs@ issue : none
bugs.kde.org number : none
applied: no
author: from trolltech

Fixes a regression in QProgress::writeToStdin()


--- src/kernel/qprocess.cpp
+++ src/kernel/qprocess.cpp
@@ -727,7 +727,7 @@ void QProcess::closeStdinLaunch()
 void QProcess::writeToStdin( const QString& buf )
 {
     QByteArray tmp = buf.local8Bit();
-    tmp.resize( tmp.size() - 1 ); // drop the implicit \0
+    tmp.resize( qstrlen( tmp.data() ) );
     writeToStdin( tmp );
 }

